azure-pipelines-pr.yml

```diff
@@ -59,34 +23,133 @@ pr:
     exclude:
     - Documentation/*
 
-# Call the pipeline-pr.yml template, which does the real work
+variables:
+- name: repoName
+  value: dotnet/wpf
+
 stages:
 - stage: build
   displayName: Build 
   jobs:
-  - template: /eng/pipeline-pr.yml
+  - template: /eng/common/templates/jobs/jobs.yml@self
     parameters:
-      ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-        runAsPublic: false
+      enablePublishBuildArtifacts: true
+      enablePublishTestResults: false # tests run in helix
```

Comment:
This says test run in helix but below I see TRX files being uploaded?  Which is it?

Reply:
I think tests run locally...not in Helix

Reply:
Right, as far as I know, there was an effort to run tests on Helix, but it wasn't completed.